test/sqlancer/dbms/TestClickHouse.java

```diff
@@ -6,19 +6,18 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import sqlancer.Main;
-import sqlancer.dbms.TestConfig;
 
 public class TestClickHouse {
 
     @Test
-    public void testClickHouseTLPWhereHaving() {
+    public void testClickHouseTLPWhereGroupBy() {
         String clickHouseAvailable = System.getenv("CLICKHOUSE_AVAILABLE");
         boolean clickHouseIsAvailable = clickHouseAvailable != null && clickHouseAvailable.equalsIgnoreCase("true");
         assumeTrue(clickHouseIsAvailable);
         Assertions.assertEquals(0,
-                Main.executeMain(new String[] { "--timeout-seconds", TestConfig.SECONDS, "--num-queries",
-                        TestConfig.NUM_QUERIES, "--num-threads", "50", "clickhouse", "--oracle", "TLPWhere", "--oracle",
-                        "TLPHaving" }));
+                Main.executeMain(new String[] { "--timeout-seconds", TestConfig.SECONDS, "--num-queries", "1000",
```

Comment:
Could you please use https://github.com/sqlancer/sqlancer/blob/e62224edcb43fe421dc87601f75090739e34e595/test/sqlancer/dbms/TestConfig.java#L4 here?

Reply:
Ok. It left after it was set to 0 with TLPHaving oracle enabled.